src/lib/service-user/service-user.factory.ts

```diff
@@ -48,6 +50,19 @@ function addImportToModule(options: ServiceOptions): Rule {
 
     tree.overwrite(options.module, content);
 
+    const nodeDependencyRef = getPackageJsonDependency(
+      tree,
+      '@techsavvyash/user-service',
```

Comment:
Check [`packages/user-service`](https://github.com/SamagraX-stencil/stencil/tree/main/packages/user-service/package.json) -- this should be mentioning the `package-name` as `@samagra-x/user-service` and hence we need to use the same.

Reply:
Noted